example/index.jsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable react/no-multi-comp, react/prop-types */
```

Comment:
As these are just examples, not sure what we want to do here. Thinking it's best to keep multiple components in one file to make this easier for newcomers to navigate. Thoughts?